speedcord/http.py

```diff
@@ -82,6 +105,19 @@ async def create_ws(self, url, *, compression) -> ClientWebSocketResponse:
         return await self.session.ws_connect(url, **options)
 
     async def request(self, route: Route, **kwargs):
+        """
+        Sends a request to the Discord API. Handles rate limits by utilizing LockManager and
```

Comment:
Great first sentence. The rest is just explaining the inner workings of this function which isn't needed at all.

Maybe change it to something like this?
```py
"""
Sends a request to the Discord API and handles ratelimits for you.
:param route: The description of the API route to send the request to
:params: All params will be send to aiohttp.request.
"""
```